docs/source/gbd2019_models/causes/ihd/index.rst

```diff
@@ -116,331 +112,224 @@ Assumptions and Limitations
-     - 
-   * - I
-     - excess mortality rate
-     - :math:`\frac{\text{deaths_cid}}{\text{population} \times \text{prevalence_cid}}`
-     - = (cause-specific mortality rate) / prevalence
-   * - I
-     - disability weights
-     - :math:`\displaystyle{\sum_{s\in \text{sequelae_cid}}} \scriptstyle{\text{disability_weight}_s \,\times\, \text{prevalence}_s}`
-     - = total disability weight over all sequelae
-   * - ALL
-     - cause specific mortality rate
-     - :math:`\frac{\text{deaths_cid}}{\text{population}}`
-     - 
+     - :math:`\sum\limits_{s\in post-mi-sequelae} \text{prevalence}_s`
+     -
+   * - A
+     - prevalence
+     - :math:`\sum\limits_{s\in acute-sequelae} \text{prevalence}_s`
+     -
+   * - S
+     - prevalence
+     - 1-prevalence_493
+     - simulants not prevalent with IHD
+   * - A2
+     - prevalence
+     - :math:`\sum\limits_{s\in angina-sequelae} \text{prevalence}_s`
+     -
+   * - S2
+     - prevalence
+     - 1-prevalence_493
+     - simulants not prevalent with IHD
 
 Transition Data
 """""""""""""""
 
-This table contains the measures needed for transition from one state to other in the cause model. The common measures used are *incident rate* to 
-move from Susceptible to Infected and *remission rate* to move from Infected to Susceptible or Recovered states. Some times there may not be transition 
-between states as in Neonatal disorders.
-
-The structure of the table is as below. 
-
 .. list-table:: Transition Data
-   :widths: 10 10 10 20 30
+   :widths: 10 10 10 10 10
    :header-rows: 1
-   
-   * - Transition
-     - Source 
-     - Sink 
-     - Value
-     - Notes
-   * - i
-     - S
-     - I
-     - 
-     - 
-   * - r
-     - I
-     - R
-     - 	
-     - 
- 
-
-An example, if the data is present in GBD,
 
-.. list-table:: Transition Data
-   :widths: 10 10 10 20 30
-   :header-rows: 1
-   
    * - Transition
-     - Source 
-     - Sink 
+     - Source State
+     - Sink State
      - Value
      - Notes
-   * - i
+   * - 1
      - S
-     - I
-     - :math:`\frac{\text{incidence_rate_cid}}{\text{1 - prevalence_cid}}`
-     - 
-   * - r
-     - I
-     - R
-     - remission_rate_cid
-     - 
-
-Sometimes, we might need to use *modelable entity id* to get data. Sometimes, we might need to calculate remission rate 
-based on average case duration. In that case, the row would look like,
-
-.. list-table:: Transition Data
-   :widths: 10 10 10 20 30
-   :header-rows: 1
-   
-   * - Transition
-     - Source 
-     - Sink 
-     - Value
-     - Notes
-   * - r
-     - I
-     - R
-     - remission_rate_cid :math:`= \frac{\text{365 person-days}}{\text{average case duration in days} \times \text{1 year}}`
-     - 
-	 
+     - A
+     - incidence_c493
+     -
+   * - 2
+     - A
+     - P
+     - 28 days
+     - duration-based transition from acute state then progress into post state
+   * - 3
+     - P
+     - A
+     - incidence_493
+     -
+   * - 4
+     - S2
+     - A2
+     - incidence_493
```

Comment:
I'm thinking that this will end up with twice as much IHD as you have estimated in GBD, because people are moving from S and S2 to IHD states.